test/player_src_equals_integration.js

```diff
@@ -72,6 +72,12 @@ describe('Player Src Equals', () => {
     expect(player.isAudioOnly()).toBeFalsy();
   });
 
+  it('allow load with startTime', async () => {
+    const startTime = parseFloat(5);
```

Comment:
nit: Don't need `parseFloat`, it is already a number.

Reply:
Changed!